code/modules/mob/living/simple_animal/revenant/revenant.dm

```diff
@@ -216,7 +216,7 @@
 	..()
 
 /datum/objective/revenant/check_completion()
-	if(!istype(owner.current, /mob/living/simple_animal/revenant) || !owner.current)
+	if(owner && !istype(owner.current, /mob/living/simple_animal/revenant) || !owner || !owner && !owner.current)
```

Comment:
@Markolie It doesn't.
Even with an or check, byond just proceeds with the if statement at the first true thing it finds.
So it would have already proceeded if owner was null, 0, or "".